proknow/ProKnow.py

```diff
@@ -30,9 +30,11 @@ class ProKnow(object):
         patients (:class:`proknow.Patients.Patients`): An instance of the Patients class.
         collections (:class:`proknow.Collections.Collections`): An instance of the Collections
             class.
+        LOCK_RENEWAL_BUFFER (int): The number of seconds to use as a buffer when renewing a lock for
+            a draft structure set.
     """
 
-    def __init__(self, base_url, credentials_file=None, credentials_id=None, credentials_secret=None):
+    def __init__(self, base_url, credentials_file=None, credentials_id=None, credentials_secret=None, LOCK_RENEWAL_BUFFER=30):
```

Comment:
Just curious, why did you decide to make `LOCK_RENEWAL_BUFFER` all caps? Is this common in python to change the case of arguments based on their "globalness/staticness"?

Reply:
I didn't look at what other libraries did. If you can find a couple counter-examples, I would consider changing it.

I made it all caps because I wanted the property to be all caps and, not having done any research on what other libraries do, I thought making the case the same between the constructor argument and the property would be a good idea.